pandas/core/frame.py

```diff
@@ -2460,11 +2460,11 @@ def to_html(
             <class 'pandas.core.frame.DataFrame'>
             RangeIndex: 5 entries, 0 to 4
             Data columns (total 3 columns):
-                #   Column     Non-Null Count  Dtype
+             #   Column     Non-Null Count  Dtype
             ---  ------     --------------  -----
-                0   int_col    5 non-null      int64
-                1   text_col   5 non-null      object
-                2   float_col  5 non-null      float64
```

Comment:
I'd accidentally introduced these extra whitespaces when I indented the docstrings as part of a previous PR, and so am fixing them now